Client/multiplayer_sa/CMultiplayerSA_Postprocess.cpp

```diff
@@ -154,6 +166,35 @@ void CMultiplayerSA::SetNightVisionEnabled(bool bEnabled, bool bNoiseEnabled)
     }
 }
 
+void CMultiplayerSA::SetUnderwaterEffectEnabled(bool isEnabled)
```

Comment:
const noexcept ?

Reply:
it modifies memory so better not use noexcept.

Use `std::uint8_t` instead though